docs/pages/examples/playground/header/control-button.tsx

```diff
@@ -23,7 +22,7 @@ export const ControlButton: FC<ControlButtonProps> = memo(
       <Menu {...menuProps}>
         <MenuButton
           as={IconButton}
-          icon={<Icon icon={faEllipsis} fontSize="1.125em" />}
+          icon={<Ellipsis fontSize="1.3em" />}
```

Comment:
I don't think it's necessary to change the fontSize.

Reply:
@fffk10

That change is intentional. The icon was too small.

Reply:
Understood, I'll solve it.